tests/helper/helper_generic.go

```diff
@@ -312,12 +317,48 @@ func CommonBeforeEach() CommonVar {
 
 // CommonAfterEach is common function that cleans up after every test Spec (It)
 func CommonAfterEach(commonVar CommonVar) {
+	// Get details, including test result for each test spec and adds it to local testResults.txt file
+	// Ginkgo test related variables.
+	commonVar.testFileName = strings.Replace(CurrentGinkgoTestDescription().FileName[strings.LastIndex(CurrentGinkgoTestDescription().FileName, "/")+1:strings.LastIndex(CurrentGinkgoTestDescription().FileName, ".")], "_", "-", -1) + ".go"
+	commonVar.testCase = CurrentGinkgoTestDescription().FullTestText
+	commonVar.testFailed = CurrentGinkgoTestDescription().Failed
+	commonVar.testDuration = CurrentGinkgoTestDescription().Duration.Seconds()
+
+	var prNum string
+	var resultsRow string
+	prNum = os.Getenv("GIT_PR_NUMBER")
+	passedOrFailed := "PASSED"
+	if commonVar.testFailed {
+		passedOrFailed = "FAILED"
+	}
+	clusterType := "OCP"
+	if IsKubernetesCluster() {
+		clusterType = "KUBERNETES"
+	}
+	testDate := strings.Split(time.Now().Format(time.RFC3339), "T")[0]
+	resultsRow = prNum + "," + testDate + "," + clusterType + "," + commonVar.testFileName + "," + commonVar.testCase + "," + passedOrFailed + "," + strconv.FormatFloat(commonVar.testDuration, 'E', -1, 64) + "\n"
+	testResultsFile := filepath.Join("/", "tmp", "testResults.txt")
```

Comment:
What are we doing with this file? I assume this is not uploaded on any cloud storage bucket.

Reply:
Not yet, once this PR gets merged I will update the pipeline script to upload it to a bucket